addons/mail/static/src/messaging/component/message_list/message_list.js

```diff
@@ -327,7 +327,7 @@ class MessageList extends Component {
                 isProcessed = true;
             } else {
                 const lastMessage = threadCache.lastMessage;
-                if (this.messageRefFromId(lastMessage.id)) {
+                if (this.messageRefFromId(lastMessage.id) && this.props.order === 'asc') {
```

Comment:
I feel like calling `_scrollToLastMessage` should work no matter the order? I'm not sure to understand the change here.

Reply:
I'll check but it wasn't working correctly at first (was like it's always scrolling at the bottom end), but I didn't do the second one-line fix when I tried

Reply:
I checked and there is a deeper problem I think.
In fact `lastMessageRef` contains the ref to the last message printed (so always the message of the bottom in the list) but sometimes it is considered as being the last message in order which are not always the same notions... For example, `scrollToLastMessage` will always scroll to the bottom of the list, so I think it's something to avoid to do it if the order is 'asc'.

We may need another notion to fix this but I think everything else seems to work correctly like this so is it really worthy ... ?

Reply:
I feel like it should be renamed to "mostRecent" instead of "last", consider it as such, and everything would be solved?

Or do we have use case where we actually want to scroll "at the bottom" no matter if it's the most or least recent?

Reply:
I don't know because the way `lastMessageRef` is computed makes that it contains really the last message printed (so the one at the bottom). So if we rename it, we should then change the way it's computed. This is fine by me, as the only places it's used, I think it should be considered as the most recent and not the last in the bottom.